website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/schema/schema-metadata-exposures.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Exposures"
+id: "metadata-schema-exposures"
+---
+
+import { ArgsTable, SchemaTable } from "./schema";
+
+The exposures object allows you to query information about all exposures in a given job. You can learn more about exposures [here](/docs/building-a-dbt-project/exposures).
```

Comment:
I like the link to docs on `exposures` from here. I think it could make sense to do the same for seeds, tests, etc. Exposures are definitely the least well-known of the bunch, but you never know who might stumble upon these docs before the others—especially if they searched for "seed" and site search landed them here.